alioth/src/virtio/dev/blk.rs

```diff
@@ -123,8 +124,10 @@ pub struct BlockConfig {
 }
 impl_mmio_for_zerocopy!(BlockConfig);
 
+#[derive(Debug, Clone, Deserialize)]
 pub struct BlockParam {
     pub path: PathBuf,
+    pub readonly: bool,
```

Comment:
please add  `#[serde(default)]` such that writable images can be simply `--blk path=/path/to/file`.